rest-api/src/main/resources/paths/auth/v4_auth_reauth.yml

```diff
@@ -0,0 +1,39 @@
+post:
+    operationId: reauthenticateV4
+    tags: ["Authentication"]
+    description: |
+        Using the reauthentication token supplied with a new session, request a new session from 
+        the server. Updates the session's timeout value, and issues a new reauthentication token. 
+        Reauthentication tokens can only be used one time. 
+    parameters:
+        - name: ReauthenticateRequest
+          in: body
+          description: Information to authenticate with the server
+          required: true
+          schema:
+                $ref: ../../definitions/reauthenticate_request.yml
+    responses:
+        200:
+            description: OK
+            schema:
+                $ref: ../../definitions/user_session_info.yml
+        401:
+            description: Authentication has failed
+            schema:
+                $ref: ../../definitions/message.yml
+        403:
```

Comment:
Is this expected to ever happen on reauth?

Reply:
I can't see how this would ever happen. We could remove it from the documentation for this reason, even though it's logically possible just from the code.